docs/en/install.md

```diff
@@ -63,7 +65,54 @@ $ kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/ma
 
 <!-- tabs:end -->
 
-## 2. Get KubeVela
+## 2. Install KubeVela Controller
+
+> Note: This should be prepared by platform builder
+
+This step will install KubeVela controller and its dependency.
+
+1. Install cert-manager
+
+    KubeVela rely on [cert-manager](https://cert-manager.io/docs/) to create certificates for webhook, if you want to
+    enable webhook, please refer to [cert-manager installation doc](https://cert-manager.io/docs/installation/kubernetes/)
+    to install one. If you already have a cert-manager in your system or just want to have a try, you can ignore this step.
```

Comment:
Step 1 跟 Step 5 的三个条件之一：
- 默认 enable webhook：需要 step 1
- disable webhook
- enable webhook，使用内置 cert-manager

step 1 里也解释了是否需要安装的条件，所以，条件看上去有点重叠和小矛盾。

或许可以这样改一下：

方式1: 
```
1. (Optional) Install cert-manager
Please refer to [cert-manager installation doc](https://cert-manager.io/docs/installation/kubernetes/) to install a cert-manager.

5. 
    By default, it will enable webhook, you must have a cert-manager installed. So you have to install one by step 1
```

方式2:
直接去掉 step 1，在 step 5 的第一个条件里说明怎么安装 cert-manager。

Reply:
make sense